### PR TITLE
Updated dependancy copyright license files.

### PR DESCRIPTION
**Pull request by seanzhougoogle**

### CODE REVIEW

**Review by greeshmaswaminathan — APPROVED** [review]

File: cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# babel-plugin-syntax-jsx
```
Comment (tivv) [inline cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT]: Does not look like COPYRIGHT

File: cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT [hunk 2]
```diff
@@ -0,0 +1,232 @@
+Copyright © 2014-2015 Cask Data, Inc.
```
Comment (tivv) [inline cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT:1]: Do we need to put our own licenses here?

File: cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Copyright Mathias Bynens <https://mathiasbynens.be/>
+
+Permission is hereby granted, free of charge, to any person obtaining
```
Comment (tivv) [inline cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT:3]: Do we have approval for this license? It does not look like a standard one.

File: cdap-distributions/src/COPYRIGHT/postcss-value-parser@4.1.0/COPYRIGHT [hunk 4]
```diff
@@ -0,0 +1,22 @@
+Copyright (c) Bogdan Chadkin <trysound@yandex.ru>
+
+Permission is hereby granted, free of charge, to any person
```
Comment (tivv) [inline cdap-distributions/src/COPYRIGHT/postcss-value-parser@4.1.0/COPYRIGHT:3]: Same here. It does not look like a standard license.

Comment (tivv) [reply cdap-distributions/src/COPYRIGHT/postcss-value-parser@4.1.0/COPYRIGHT:3]: Is it MIT?

File: cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT (hunk 3 above)
Comment (tivv) [reply cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT:3]: Is it MIT?

File: cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT (hunk 1 above)
Comment (seanzhougoogle) [reply cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT]: good catch, it's generated by scripts, this looks like an UI dependency. Let me see how it looks like for other releases.

File: cdap-distributions/src/COPYRIGHT/postcss-value-parser@4.1.0/COPYRIGHT (hunk 4 above)
Comment (sechegaray) [reply cdap-distributions/src/COPYRIGHT/postcss-value-parser@4.1.0/COPYRIGHT:3]: Here it mentions it's MIT: https://www.npmjs.com/package/postcss-value-parser

File: cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT (hunk 3 above)
Comment (sechegaray) [reply cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT:3]: Listed as MIT: https://www.npmjs.com/package/jsesc

File: cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT (hunk 2 above)
Comment (seanzhougoogle) [reply cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT:1]: see :  https://github.com/cdapio/cdap/pull/13722#discussion_r738600982 :)

File: cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT (hunk 3 above)
Comment (seanzhougoogle) [reply cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT:3]: we already have such license for 6.5.1 : https://github.com/cdapio/cdap/blob/release/6.5/cdap-distributions/src/COPYRIGHT/punycode@1.4.1/COPYRIGHT

File: cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT (hunk 2 above)
Comment (tivv) [reply cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT:1]: IMHO it's weird, but who am I to question traditions :)

File: cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT (hunk 3 above)
Comment (tivv) [reply cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT:3]: Good thing it's listed as MIT.

File: cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT (hunk 1 above)
Comment (seanzhougoogle) [reply cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT]: I know why it looks like this now, that's the whole content of its README file...

Comment (seanzhougoogle) [reply cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT]: it's MIT license too : https://www.npmjs.com/package/@babel/plugin-syntax-jsx

Comment (tivv) [reply cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT]: This link does not show MIT, but could you put it into overrides with a link that clarifies it's MIT?

Comment (seanzhougoogle) [reply cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT]: updated.

**Review by tivv — APPROVED** [review]

**Pull request merged**